src/main/java/com/spotify/autoscaler/db/PostgresDatabase.java

```diff
@@ -439,4 +460,23 @@ public boolean setMinNodesOverride(
   public int getTotalConnections() {
     return dataSource.getHikariPoolMXBean().getTotalConnections();
   }
+
+  @Override
+  public int deleteBigtableClusters(final String projectId, final String instanceId) {
+    final String sql = "DELETE FROM autoscale WHERE project_id = ? AND instance_id = ?";
+    return jdbc.getJdbcOperations().update(sql, projectId, instanceId);
+  }
+
+  @Override
+  public int deleteBigtableClustersExcept(
```

Comment:
This method sounds so specific, and I wonder if we can do this a composition of the other methods